Makefile

```diff
@@ -10,3 +10,5 @@ appimage:
 	mv nvim.appimage ./build/nvim
 	nvim --headless --noplugin -u scripts/minimal.vim -c 'PlenaryBustedDirectory tests/plenary/ tests/minimal_init.vim'
 
+generate_filetypes:
+	nvim --headless -c 'luafile scripts/update_filetypes_from_github.lua' -c 'qa!'
```

Comment:
Do we want a ci job for that? Could be cool.
Also i think i will open a new PR for the current pipeline, which removes the appimage from root and adds `luacheck`